storagenode/monitor/monitor.go

```diff
@@ -127,6 +124,11 @@ func (service *Service) Run(ctx context.Context) (err error) {
 		if err != nil {
 			service.log.Error("error during updating node information: ", zap.Error(err))
 		}
+
+		err = service.RecalculateSpaceUsedLive(ctx)
```

Comment:
It looks like this is the only place that calls `store.RecalculateSpaceUsedCache()` (by way of `service.RecalculateSpaceUsedLive()`). I don't think the monitor service should be the one responsible for managing the freshness of the cache inside the piece store. For one thing, the piece store service should continue working correctly even if there isn't a monitor service running at all. Maybe we should have a new `sync2.Cycle` object owned by the store-wrapper object itself (or maybe owned by whatever object ends up owning the cache object, if that structure changes).